src/libs/highlevel/README.md

```diff
@@ -153,8 +153,9 @@ and therefore has no specified default value.
 The handler will also be called whenever you pass `NULL` where a function expects an `ElektraError **`. In this case the error code will be
 `ELEKTRA_ERROR_CODE_NULL_ERROR`.
 
-The default callback simply logs the error with `ELEKTRA_LOG_DEBUG` and then calls `exit()` with the error code of the error. It is highly
-recommended you either use `atexit()` in you application or set a custom callback, to make sure you won't leak memory.
```

Comment:
At exit memory is cleaned up anyway? Why this recommendation?

Reply:
"won't leak memory" should be "dispose of all resources". The application might e.g. use sockets that need to be closed. Even if we assume the user knows that, we should still mention the the default callback calls `exit()`.